pulseapi/tests.py

```diff
@@ -164,6 +164,8 @@ class PulseMemberTestCase(TestCase):
     """
     A test case wrapper for "plain users" without any staff or admin rights
     """
+    maxDiff = None
```

Comment:
I made this small change to make error logging during tests a bit more informative